isaric/parsers/ccp-uganda.toml

```diff
@@ -0,0 +1,895 @@
+combinedType = "any"
+fields = [
+    { field = "overall_invasive_proccur", values = { 1 = true, 0 = false, 3 = false } },
+    { field = "inv_vent", values = { 1 = true, 0 = false } }
+]
+
+[visit.treatment_antivirals]
+field = "meds_given"
+values = { 1 = false, 2 = false, 3 = false, 4 = false, 5 = true, 6 = false, 7 = false, 8 = false, 9 = false }
+
+[visit.treatment_antibiotics]
+combinedType = "any"
+fields = [
+    { field = "meds_given", values = { 1 = false, 2 = false, 3 = false, 4 = false, 5 = false, 6 = true, 7 = false, 8 = false, 9 = false } },
+    { field = "daily_antibiotic", ref = "Y/N/NK" },
+    { field = "antibio_meds_oth", apply = { function = "isNotNull" } },
+    { field = "sup_antibiotic_agent", apply = { function = "isNotNull" } }
+]
+
+[visit.treatment_antibiotics_type]
+  combinedType = "set"
+  excludeWhen = "none"
+  fields = [
+    { field = "antibio_meds_oth" },
+    { field = "sup_antibiotic_agent" }
+  ]
+
+[visit.treatment_anticoagulation]
+field = "meds_given"
+values = { 1 = false, 2 = false, 3 = false, 4 = true, 5 = false, 6 = false, 7 = false, 8 = false, 9 = false }
+
+[visit.treatment_inhaled_nitric_oxide]
+combinedType = "any"
+fields = [
+    { field = "overall_nitrcoxid", values = { 1 = true, 0 = false, 3 = false } },
+    { field = "nitric_oxide", values = { 1 = true, 0 = false } }
+]
+
+[visit.treatment_noninvasive_ventilation]
+combinedType = "any"
+fields = [
+    { field = "overall_noninvasive_proccur", values = { 1 = true, 0 = false, 3 = false } },
+    { field = "non_inv_vent", values = { 1 = true, 0 = false } }
+]
+
+[visit.treatment_antimalarial]
+field = "daily_antimal"
+ref = "Y/N/NK"
+
+[visit.treatment_high_flow_nasal_cannula]
+field = "can_oxy_ther"
+values = { 1 = true, 0 = false }
+
+[visit.treatment_intravenous_fluids]
+field = "daily_iv_fluids"
+ref = "Y/N/NK"
+
+[visit.treatment_neuraminidase]
+field = "sup_meds_antivirals"
+values = { 1 = false, 2 = false, 3 = false, 4 = false, 5 = true, 6 = false }
+
+[visit.treatment_neuromuscular_blocking_agents]
+field = "neuro_block"
+values = { 1 = true, 0 = false }
+
+[visit.treatment_other]
+combinedType = "set"
+excludeWhen = "none"
+fields = [
+    { field = "oth_interv_sp", description = "Specify Other intervention or procedure?" }
+]
+
+[visit.outcome]
+field = "outcome"
+values = { 1 = "Alive at discharge", 2 = "Death", 3 = "Ran away/unknown", 4 = "Referred/Transferred" }
+
+[visit.date_outcome]
+field = "outcome_date"
+
+
+# OBSERVATION
+# required fields - phase, date, name
+
+[[observation]] 
+name = "avpu"
+date = { field = "signs_symps_date" }
+phase = "admission"
+text = { field = "avpu", values = { 1 = "ALERT", 2 = "Only responsive to VOICE", 3 = "Only responsive to PAIN", 4 = "UNRESPONSIVE" } }
+
+[[observation]] 
+name = "avpu"
+date = { field = "signs_symps_date" }
+phase = "admission"
+text = { field = "avpu_study", values = { 1 = "ALERT", 2 = "Only responsive to VOICE", 3 = "Only responsive to PAIN", 4 = "UNRESPONSIVE" } }
```

Comment:
Is this the right date for an admission phase symptom? surely it should be the hierarchy? Same for all observations

Reply:
the description for 'signs_symps_date' in the data dictionary is "signs_and_symptoms_on_admission_day", and this refers to a date ..